static/app/views/issueList/savedIssueSearches.tsx

```diff
@@ -0,0 +1,168 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import partition from 'lodash/partition';
+
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import space from 'sentry/styles/space';
+import {Organization, SavedSearch} from 'sentry/types';
+
+interface SavedIssueSearchesProps {
+  isOpen: boolean;
+  onSavedSearchSelect: (savedSearch: SavedSearch) => void;
+  organization: Organization;
+  savedSearch: SavedSearch | null;
+  savedSearchLoading: boolean;
+  savedSearches: SavedSearch[];
+}
+
+interface SavedSearchItemProps {
+  onSavedSearchSelect: (savedSearch: SavedSearch) => void;
+  savedSearch: SavedSearch;
+}
+
+const SavedSearchItem = ({onSavedSearchSelect, savedSearch}: SavedSearchItemProps) => {
+  return (
+    <SearchListItem>
+      <StyledItemButton
+        aria-label={savedSearch.name}
+        onClick={() => onSavedSearchSelect(savedSearch)}
+      >
+        <div>
+          <SavedSearchItemTitle>{savedSearch.name}</SavedSearchItemTitle>
+          <SavedSearchItemDescription>{savedSearch.query}</SavedSearchItemDescription>
+        </div>
+      </StyledItemButton>
+    </SearchListItem>
+  );
+};
+
+const SavedIssueSearches = ({
+  organization,
+  isOpen,
+  onSavedSearchSelect,
+  savedSearchLoading,
+  savedSearches,
+}: SavedIssueSearchesProps) => {
+  if (!isOpen) {
+    return null;
+  }
+
+  if (!organization.features.includes('issue-list-saved-searches-v2')) {
+    return null;
+  }
+
+  if (savedSearchLoading) {
+    return (
+      <StyledSidebar>
+        <LoadingIndicator />
+      </StyledSidebar>
+    );
+  }
+
+  const [recommendedSavedSearches, orgSavedSearches] = partition(
+    savedSearches,
+    item => item.isGlobal
+  );
```

Comment:
We may also want to filter out `isPinned` as well?

Reply:
pinned save searches do come back from the API

Reply:
Good catch! Will filter those out